noxfile.py

```diff
@@ -95,6 +95,8 @@ def tests(session):
     # version didn't fix the `2to3` issue when building Python ICAT, perhaps because
     # Python ICAT isn't built on the downgraded version for some reason?
     session.run("poetry", "run", "pip", "uninstall", "-y", "setuptools")
```

Comment:
For consistency, you might as well remove the poetry run from the uninstall - I did this on my machine without issue

Reply:
Good point, I removed it.